bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/accessories/HomekitCharacteristicFactory.java

```diff
@@ -379,12 +378,8 @@ private static BrightnessCharacteristic createBrightnessCharacteristic(final Hom
             return CompletableFuture.completedFuture(value);
         }, (brightness) -> {
             final Item oItem = item.getItem();
```

Comment:
My comment regarding the naming is also valid here. It would be better to have somthing like 

```
final Item item = taggedItem.getItem();
```